Sourcery/Parsing/FileParser.swift

```diff
@@ -533,11 +533,14 @@ extension FileParser {
 
     internal func parseParameter(_ source: [String: SourceKitRepresentable]) -> MethodParameter? {
         guard let (name, _, _) = parseTypeRequirements(source),
-            let type = source[SwiftDocKey.typeName.rawValue] as? String else { return nil }
+              let type = source[SwiftDocKey.typeName.rawValue] as? String else {
+            return nil
+        }
 
+        let inOut = type.hasPrefix("inout ")
```

Comment:
> Can inout be mixed with any other attributes that can be used for method parameters?
I thought about it and quick Googling didn't show anything. So I think for the first version we can keep it as just a boolean.

And I've also thought about putting it into `TypeName` and I have doubts whether it makes sense to put it there. As far as I know the `inout` could be the only method parameter in Swift language, if so - from my point of view it's only related to the method and to parameter itself, rather than the type of the parameter. But I would like to hear your opinion about it 😄